### PR TITLE
Augment stdout of import command

### PR DESCRIPTION
**Pull request by Pytal**
See title 👆

### CODE REVIEW

File: lib/Command/Import.php [hunk 1]
```diff
@@ -72,33 +68,32 @@ protected function configure(): void {
...
+			$io->writeln("Importing from ${path}…");
 			$importSource = new ImportSource($path);
-			$this->migrationService->import($importSource, $user, $output);
-			$output->writeln("Successfully imported from ${path}");
+			$this->migrationService->import($importSource, $user, $io);
+			$io->writeln("Successfully imported from ${path}");
 		} catch (\Exception $e) {
-			$output->writeln("$e");
-			$output->writeln("<error>" . $e->getMessage() . "</error>");
+			$io->error($e->getMessage());
```
Comment (come-nc) [inline lib/Command/Import.php:96]: Did you remove the trace on purpose?

**Review by come-nc — COMMENTED** [review]
Did you test that `--no-interaction` option is still working?

File: lib/Command/Import.php (hunk 1 above)
Comment (Pytal) [reply lib/Command/Import.php:96]: Yes, seemed too verbose but if you think we should add the trace output back here and in the export command then we could add it back?

**Comment by Pytal** [comment]
> Did you test that `--no-interaction` option is still working?

Yes, still works and defaults to no

File: lib/Command/Import.php (hunk 1 above)
Comment (come-nc) [reply lib/Command/Import.php:96]: Maybe add it in high verbose level then?

**Review by come-nc — APPROVED** [review]

**Pull request merged**